ci/ci.el7/rpmtest/test-rpm-install.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -ex -o pipefail
+
+CID=
+docker_rm() {
+  if [[ -z "$CID" ]]; then
+      return 0
+  fi
+  if [[ -n "$LEAVE_CONTAINER" ]]; then
+      if [[ "${LEAVE_CONTAINER}" != "0" ]]; then
+          echo "Skip to clean container: ${CID}"
+          return 0
+      fi
+  fi
+  docker rm -f "${CID}"
+}
+trap 'docker_rm' EXIT
+
+BUILD_ENV_PATH=${1:?"ERROR: env file is not given."}
+if [[ -n "${BUILD_ENV_PATH}" && ! -f "${BUILD_ENV_PATH}" ]]; then
+  echo "ERROR: Can't find the file: ${BUILD_ENV_PATH}" >&2
+  exit 1
+fi
+
+set -a
+. ${BUILD_ENV_PATH}
+set +a
+
+if [[ -n "$JENKINS_HOME" ]]; then
+  # openvnet-axsh/branch1/el7
+  img_tag="openvnet/rpmtest:${BRANCH}.${RELEASE_SUFFIX}"
+else
+  img_tag="rpm-install.openvnet.$(git rev-parse --abbrev-ref HEAD).${BUILD_OS}"
+fi
+
+docker build \
+       --build-arg BRANCH="${BRANCH}" \
+       --build-arg RELEASE_SUFFIX="${RELEASE_SUFFIX}" \
+       --build-arg BUILD_URL="${BUILD_URL}" \
+       --build-arg ISO8601_TIMESTAMP="$(date -u +"%Y-%m-%dT%H:%M:%SZ")" \
+       --build-arg LONG_SHA="${LONG_SHA}" \
+       -t "${img_tag}" -f "./ci/ci.el7/rpmtest/Dockerfile" .
+
+CID=$(docker run -d ${BUILD_ENV_PATH:+--env-file $BUILD_ENV_PATH} -d "${img_tag}")
```

Comment:
This rpmtest stage should be removed. There is no point in installing RPM packages only to immediately throw away the environment without checking if anything worked. The integration test will catch any problems there are with the packages.